german/the-black-hack.md

```diff
@@ -259,8 +260,8 @@ Traditionelle OSR-Spiele bilden magische Gegenstände ab, indem sie +1 oder +2 a
 
 **Start-TP :** W10 + 4
 **TP pro Stufe/Erholung :** 1W10
-**Waffen & Rüstung :** Alle Waffen und Rüstungen
-**Angriffsschaden :** 1W8 / 1W6 Unbewaffnet oder Improvisiert
+**Waffen & Rüstung :** alle Waffen und Rüstungen
+**Angriffsschaden :** 1W8 / 1W6 unbewaffnet oder improvisiert
```

Comment:
Ich muss sagen, dass ich wirklich wenige deutsche Regelwerke lese. Im Englischen ist es oft üblich, Regelbegriffe groß zu schreiben oder anders hervorzuheben. Ich finde das oft hilfreich, um daran erinnert zu werden, ei en Satz im Regelzusammenhang zu lesen.

Wenn ich z.B. in einem Abenteuer schreibe: "Im Geräteschuppen können die Helden **improvisierte** Waffen finden.", dann hebt das hervor, dass für diese Gegenstände im Kampf dann der verringerte Schaden gilt. Im Englischen klappt das mit Großschreibung halt gut, weil die da seltener vorkommt.